.circleci/config.yml

```diff
@@ -192,6 +204,26 @@ jobs:
           command: xcodebuild -project MapboxDirections.xcodeproj -scheme 'MapboxDirections watchOS' -destination 'platform=watchOS Simulator,name=Apple Watch Series 5 - 44mm,OS=<< parameters.watchOS >>' clean build
       - save-cache
 
+  publish-documentation:
+    macos:
+      xcode: "12.5.1"
+    steps:
+      - checkout
+      - install-mapbox-token
+      - install-carthage
+      - install-mbx-ci
+      - install-gh
+      - run:
+          name: "Publish Documentation"
+          command: |
+            ./scripts/publish-documentation.sh << pipeline.git.tag >>
```

Comment:
I would recommend splitting this step into several separate ones:
* Build Documentation
* Setup Repo Write Access
* Create Docs PR